### DISCUSSION TITLE
[General] Kubernetes support

### DISCUSSION THREAD

**Discussion by ghost** [zkSync-Community-Hub/zksync-developers#689] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Is there any official zkSync Kubernetes helm chart/glasskube/kustomize, like this [docker-compose](https://github.com/matter-labs/local-setup/blob/main/zk-chains-docker-compose.yml) file for local development or testnet setup?

**Comment by alhenaai** [comment]
Thank you for your question! I’m unable to provide an answer at this moment, but I have forwarded your inquiry to our team. Someone will be with you shortly to assist further. Your patience is appreciated!

**Reply by bxpana** [reply, marked as answer]
From my understanding there is no official ZKsync Kubernetes helm chart/glasskube/kustomize at this moment, but will turn this into a Feature Request to be able to come back to

**Reply by g3s1n3** [reply]
Hej, do you happen to have a feature request for this so we can track this? It would be a great feature for teams going into production with ZKsync. Thanks!
